.changeset/metal-fishes-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-app-api': patch
+---
+
+Updated AppManager to return an absolute URL for `backend.baseUrl` when a relative URL is provided.
```

Comment:
AppManager is an internal concern, we'd need to work this so that it makes sense to users of the package. Perhaps just something about how apps will now detect when a relative backend baseUrl is provided and will rewrite the config accordingly?